@ecomplus/storefront-app/src/components/html/EcAddressForm.html

```diff
@@ -48,7 +48,7 @@
     </div>
   </div>
 
-  <slide-y-up-transition>
+  <transition-group enter-active-class="animated fadeInDown">
```

Comment:
```suggestion
  <transition enter-active-class="animated fadeInDown">
```

Como tem só um elemento filho, pode ser só um `transition` em vez de `transition-group`